postgres/init/20-portal.sql

```diff
@@ -1,2066 +1,2100 @@
+    firstname character varying(255),
+    lastlogin bytea,
+    lastname character varying(255),
+    company_id uuid NOT NULL,
+    company_user_status_id integer NOT NULL
+);
+
+
+--
+-- Name: connector_statuses; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.connector_statuses (
+    id integer NOT NULL,
+    label character varying(255) NOT NULL
+);
+
+
+--
+-- Name: connector_types; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.connector_types (
+    id integer NOT NULL,
+    label character varying(255) NOT NULL
+);
+
+
+--
+-- Name: connectors; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.connectors (
+    id uuid NOT NULL,
+    name character varying(255) NOT NULL,
+    connector_url character varying(255) NOT NULL,
+    type_id integer NOT NULL,
+    status_id integer NOT NULL,
+    provider_id uuid NOT NULL,
+    host_id uuid,
+    location_id character(2) NOT NULL
+);
+
+
+--
+-- Name: consent_statuses; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.consent_statuses (
+    id integer NOT NULL,
+    label character varying(255) NOT NULL
+);
+
+
+--
+-- Name: consents; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.consents (
+    id uuid NOT NULL,
+    date_created timestamp with time zone NOT NULL,
+    comment character varying(255),
+    consent_status_id integer NOT NULL,
+    target character varying(255),
+    agreement_id uuid NOT NULL,
+    company_id uuid NOT NULL,
+    document_id uuid,
+    company_user_id uuid NOT NULL
+);
+
+
+--
+-- Name: countries; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.countries (
+    alpha2code character(2) NOT NULL,
+    alpha3code character(3),
+    country_name_de character varying(255) NOT NULL,
+    country_name_en character varying(255) NOT NULL
+);
+
+
+--
+-- Name: document_status; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.document_status (
+    id integer NOT NULL,
+    label character varying(255) NOT NULL
+);
+
+
+--
+-- Name: document_templates; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.document_templates (
+    id uuid NOT NULL,
+    date_created timestamp with time zone NOT NULL,
+    date_last_changed timestamp with time zone,
+    documenttemplatename character varying(255) NOT NULL,
+    documenttemplateversion character varying(255) NOT NULL
+);
+
+
+--
+-- Name: document_types; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.document_types (
+    id integer NOT NULL,
+    label character varying(255) NOT NULL
+);
+
+
+--
+-- Name: documents; Type: TABLE; Schema: portal; Owner: -
+--
+
+CREATE TABLE portal.documents (
+    id uuid NOT NULL,
+    date_created timestamp with time zone NOT NULL,
+    document_name character varying(255) NOT NULL,
+    document_type_id integer,
+    company_user_id uuid,
+    document_hash bytea,
+    document_content bytea DEFAULT '\x'::bytea NOT NULL,
```

Comment:
I notice we do not define default-values in any other table (besides company_assigned_apps), so here we should not do this here to be consistent.

Reply:
i removed the default value.